### PR TITLE
[Fix #24] add wrapper for CLI help print; added test (WIP)

### LINKED ISSUE #24: Fix logging
Currently generating:
```
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.sun.xml.bind.v2.runtime.reflect.opt.Injector (file:/home/blcha/projects/20doprava-ucl/git/bpmn-convertor-examples/.target/bpmn2stamp.jar) to method java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int)
WARNING: Please consider reporting this to the maintainers of com.sun.xml.bind.v2.runtime.reflect.opt.Injector
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
/tmp/mapping-tmp-16378272493106498894.map
/home/blcha/projects/20doprava-ucl/git/bpmn-convertor-examples/processes/ucl/dozor-nad-provozovateli-letist-bbo.ttl
http://onto.fel.cvut.cz/ontologies/processes/ucl/dozor-nad-provozovateli-letist-bpmn
file:/home/blcha/projects/20doprava-ucl/git/bpmn-convertor-examples/processes/ucl/dozor-nad-provozovateli-letist-bbo.ttl
http://onto.fel.cvut.cz/ontologies/processes/ucl/dozor-nad-provozovateli-letist-bpmn
/tmp/mapping-tmp-16483033631579641637.map
/home/blcha/projects/20doprava-ucl/git/bpmn-convertor-examples/processes/ucl/dozor-nad-provozovateli-letist-bbo.ttl
http://onto.fel.cvut.cz/ontologies/processes/ucl/dozor-nad-provozovateli-letist-bpmn

```

A/C:
- logs should make sense and should have a level (debug/info)
- we should understand what happened (all files created should be mentioned)

### PR DESCRIPTION
**Pull request by grigobog**

### CODE REVIEW

File: bpmn2stamp-console/src/test/java/cz/cvut/kbss/bpmn2stamp/console/ConsoleRunnerTest.java [hunk 1]
```diff
@@ -2,28 +2,75 @@
...
+		this.bboTypeArgsProcessor = Mockito.mock(BboTypeArgsProcessor.class);
+		this.stampFromBboTypeArgsProcessor = Mockito.mock(StampFromBboTypeArgsProcessor.class);
+		this.converterService = Mockito.mock(Bpmn2StampConverterService.class);
+	}
 
-		Assertions.assertThatThrownBy(() -> {
-			new ConsoleRunner().run(new String[]{"-t", "bbo", "-iri", "testBaseIri", "-ibpmn", "testBpmnInput", "-iorg", "testOrgInput", "-iam", "testActorInput", "-out", "testOutput"});
-		}).isNotInstanceOf(ParseException.class);
+	@Test
+	public void mainTest() throws ParseException {
```
Comment (blcham) [inline bpmn2stamp-console/src/test/java/cz/cvut/kbss/bpmn2stamp/console/ConsoleRunnerTest.java:34]: Not sure what this test supposed to do

**Review by blcham — APPROVED** [review]
LGTM,can be merged without change, although I would remove the mentioned test if it does not do anything meaningful (there is a lot of commented code)

File: bpmn2stamp-console/src/test/java/cz/cvut/kbss/bpmn2stamp/console/ConsoleRunnerTest.java (hunk 1 above)
Comment (grigobog) [reply bpmn2stamp-console/src/test/java/cz/cvut/kbss/bpmn2stamp/console/ConsoleRunnerTest.java]: This is unfinished WIP test, finished in the following commit

**Review by blcham — APPROVED** [review]
LGTM

**Pull request merged**